### DISCUSSION TITLE
Disable namespace autocomplet for refernces

### DISCUSSION THREAD

**Discussion by jang1er** [James-Yu/LaTeX-Workshop#4117] [Q&A]
Hello,
I am currently working on my university group project, and we are using VS Code with docker to edit and make our PDF files.
We have a few custom commands to generate tables for functional requirements and to link them together.
 A small hindrance I encountered is that intellisense suggests references with the namespace (e.g.: fr:Server Functions). I have to manually delete this namespace for it to properly link in the PDF. Is there any option for it to be turned off?

**Comment by jlelong** [comment]
I am not sure to fully understand your problem. Could you provide a small example?

**Reply by jang1er** [reply]
I realized this only happens based on our setup of references. We have changed the system and now it works flawlessly.
